src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -102,12 +102,15 @@ public Person toModelType() throws IllegalValueException {
             throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT,
                     Telegram.class.getSimpleName()));
         }
-        if (handle != null && !Telegram.isValidTelegram(handle)) {
+        if (!Telegram.isValidTelegram(handle)) {
```

Comment:
Based on latest discussion, are the optional and mandatory fields as follows:
Phone, Email, Github, Tags are optional
Name, Telegram are mandatory?

(But here, we are only throwing an error only if GitHub username is null?)

Reply:
Okay thanks, I made the changes. 

Also instead of saving optional fields as `null` into `Person.java`, I made it to save as for example, `new Github(null)` as saving it as `null` led to many `NullPointerException`s.